src/components/feedback-widget/feedback-widget.config.yml

```
@@ -0,0 +1,16 @@
+label: Feedback widget
+default: success
+
+context:
+  widgetVersion: "latest"
+  feedbackApiUrl: "https://innovation.nj.gov/app/feedback/dev"
+  mockResponse: { status: 200, body: {message: "Success", feedbackId: 1} }
```

dust - For consistency, please add spaces around the curly brackets
```suggestion
  mockResponse: { status: 200, body: { message: "Success", feedbackId: 1 } }
```